Submodule oofatfs contains modified content
diff --git a/oofatfs/src/ff.c b/oofatfs/src/ff.c
index 171b599..1925bde 100644
--- a/oofatfs/src/ff.c
+++ b/oofatfs/src/ff.c
@@ -5392,7 +5392,7 @@ FRESULT f_mkfs (
     UINT len            /* Size of working buffer [byte] */
 )
 {
-    const UINT n_fats = 1;      /* Number of FATs for FAT/FAT32 volume (1 or 2) */
+    const UINT n_fats = 2;      /* Number of FATs for FAT/FAT32 volume (1 or 2) */
     const UINT n_rootdir = 512; /* Number of root directory entries for FAT volume */
     static const WORD cst[] = {1, 4, 16, 64, 256, 512, 0};  /* Cluster size boundary for FAT volume (4Ks unit) */
     static const WORD cst32[] = {1, 2, 4, 8, 16, 32, 0};    /* Cluster size boundary for FAT32 volume (128Ks unit) */
diff --git a/oofatfs/src/ffconf.h b/oofatfs/src/ffconf.h
index 036b6f0..ea53ff8 100644
--- a/oofatfs/src/ffconf.h
+++ b/oofatfs/src/ffconf.h
@@ -199,7 +199,7 @@
 
 
 #define FF_MIN_SS       512
-#define FF_MAX_SS       512
+#define FF_MAX_SS       4096
 /* This set of options configures the range of sector size to be supported. (512,
 /  1024, 2048 or 4096) Always set both 512 for most systems, generic memory card and
 /  harddisk. But a larger value may be required for on-board flash memory and some
